### PR TITLE
[SPI][Task]add SQL Task

### PR DESCRIPTION
**Pull request by CalvinKirs**
Add SQL Task And Fix Task Parse Err

### CODE REVIEW

**Review by zhuangchong — DISMISSED** [review]
+1

File: dolphinscheduler-spi/src/main/java/org/apache/dolphinscheduler/spi/common/Constants.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+package org.apache.dolphinscheduler.spi.common;
+
+public class Constants {
+
+    /**
+     * task log info format
+     */
+    public static final String TASK_LOG_INFO_FORMAT = "TaskLogInfo-%s";
```
Comment (zhuangchong) [inline dolphinscheduler-spi/src/main/java/org/apache/dolphinscheduler/spi/common/Constants.java]: This variable exists in org.apache.dolphinscheduler.spi.task.TaskConstants

Comment (CalvinKirs) [reply dolphinscheduler-spi/src/main/java/org/apache/dolphinscheduler/spi/common/Constants.java]: done, thx for your remind

**Review by zhuangchong — APPROVED** [review]
+1

**Pull request merged**